html/install.php

```diff
@@ -4,7 +4,7 @@
 session_start();
 if (empty($_POST) && !empty($_SESSION) && !isset($_REQUEST['stage'])) {
     $_POST = $_SESSION;
-} elseif (!file_exists("config.php")) {
+} elseif (!file_exists("config.php") && !file_exists('../config.php')) {
```

Comment:
You should not use the relative directory.
try:
```php
$librenms_dir = realpath(__DIR__ . '/..');
...
} elseif (!file_exists("$librenms_dir/config.php")) {
```

Reply:
Changed accordingly